array-methods.js

```diff
@@ -104,17 +161,22 @@ var areStatesInHigherStateSum = null;
   have a sum of account values greater than 2,550,000
   otherwise set it to be `false`
  */
-var anyStatesInHigherStateSum = null;
-
+const anyStatesInHigherStateSum = Object.entries(stateSums)
+  .filter(function (arr) {
```

Comment:
Just a note, the filter here does not need to be on the next line but I have seen in both ways in the wild.